server.js

```diff
@@ -0,0 +1,7 @@
+var express = require('express')
```

Comment:
Consider using `const` instead of `var` for requiring modules as these references are not expected to change. [important]